index.md

```diff
@@ -52,3 +192,57 @@ which most in the community are members of.
 If you have technical questions around DNS flag day you can join
 [the DNS-operations mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-operations)
 and ask them there.
+
+Get in touch
+============
+
+For press & media inquiries please use media (at) dns-oarc.net and please put
+“DNS Flag Day" in the email subject line.
+
+- Web https://dnsflagday.net/
+- Twitter https://twitter.com/dnsflagday
+- Announcements: https://lists.dns-oarc.net/mailman/listinfo/dns-announce
+- Discussion: https://lists.dns-oarc.net/mailman/listinfo/dns-operations
+
+FAQ
+===
+
+- Q: Is DNS over UDP dead?
+
+  A: No, UDP will still be the main means of transportation.
+
+- Q: TL;DR [RFC 7766](https://tools.ietf.org/html/rfc7766)
+
+  A: DNS **MUST** work over TCP!
+
+- Q: Will everything break on date-to-be-decided 2020?
+
+  A: **No!** The date that will be announced is just a marker in time from
+     which the default behavior _**in new software releases**_ will be
+     changed.
+
+- Q: Why is TCP support so important?
+
+  A: TCP support should be implemented from the beginning of DNS, although some
```

Comment:
```suggestion
  A: TCP support should be implemented from the beginning, although some
```
@matje `of DNS` sounds a bit weird, what you think?

Reply:
from the beginning of what? ;-)

But fair enough, happy to see the suggested change being merged.

Reply:
@matje how about now?